app/page.tsx

```diff
@@ -1,7 +1,15 @@
+"use client";
+
+import SearchBar from "@/components/SearchBar";
+import React from "react";
+
 export default function Home() {
+  const [albumSearch, setAlbumSearch] = React.useState<string>("");
+
   return (
-    <main>
-      <h1>Home Page</h1>
+    <main className="flex flex-col items-center justify-center ">
+      <SearchBar onChange={(value: string) => setAlbumSearch(value)} />
```

Comment:
no need to explicitly define this, TS knows

```suggestion
      <SearchBar onChange={(value) => setAlbumSearch(value)} />
```